exam_2/src/utils/translate/get_api_url.ts

```diff
@@ -0,0 +1,9 @@
+import {api_key, api_url} from './constants'
+import * as T from './types'
+
+export default function get_api_url (param: T.ITranslate): any {
```

Comment:
Возвращается строка?

```suggestion
export default function get_api_url (param: T.ITranslate): string {
```